apps/ligo-virgo/src/pages/tools/icons/Icons.tsx

```diff
@@ -1,9 +1,15 @@
-import { type FC, useRef } from 'react';
+import { type ComponentType, useRef } from 'react';
```

Comment:
please provide a conversion script - for example, saved in tools/ and replaced by core contributors, too large pr will make it difficult for us to review

Reply:
> please provide a conversion script 

> Using [jscodeshift](https://github.com/facebook/jscodeshift) and [codemod-replace-react-fc-typescript](https://github.com/gndelia/codemod-replace-react-fc-typescript)to remove React.FC from components.

Reply:
great tools, we've cleaned up the code with it